crates/client/src/lib.rs

```diff
@@ -39,7 +40,7 @@ pub struct Client<P, C> {
     api: api::Client,
 }
 
-impl<P: PackageStorage, C: ContentStorage> Client<P, C> {
+impl<P: RegistryStorage, C: ContentStorage> Client<P, C> {
```

Comment:
There's a few places here where comments refer to "package" storage that should be updated to "registry storage".

Reply:
Also, should rename the `packages` method (perhaps simply to `registry`?).